src/handle-config.js

```diff
@@ -0,0 +1,34 @@
+const fs = require('fs');
+const path = require('path');
+const util = require('node:util');
+const exec = util.promisify(require('node:child_process').exec);
+
+const logger = require('./logger');
+
+const TMP = '/tmp';
+const CONFIG_FILE = 'pages.json';
```

Comment:
Should we also look for `federalist.json` for legacy config

Reply:
My proposal above is basically that we don't check for the old file. This will be an opt-in, "update your site/build" style change so I like the idea of moving folks to more pages specific configuration

Reply:
I agree with @drewbo's leaning on on this.

Reply:
Sounds good.